tests/bats/deploy/_helpers_deploy.bash

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+load ../_helpers
+load ../_bats-mock
+
+setup() {
```

Comment:
this is the magic that makes mocked binaries work (not as easy as it should be out of the box)